accessibility-checker.php

```diff
@@ -1069,19 +1065,12 @@ function edac_anww_update_post_meta() {
 
 	$option_name = 'edac_anww_update_post_meta';
 
-	if ( get_option( $option_name ) === false && EDAC_ANWW_ACTIVE === true ) {
-
-		add_option( $option_name, true );
-
-		edac_update_post_meta( 'link_blank' );
-
-	} elseif ( get_option( $option_name ) === true && EDAC_ANWW_ACTIVE === false ) {
-
+	if ( get_option( $option_name ) === false && EDAC_ANWW_ACTIVE ) {
+		update_option( $option_name, true );
+	} elseif ( get_option( $option_name ) === true && ! EDAC_ANWW_ACTIVE ) {
 		delete_option( $option_name );
-
-		edac_update_post_meta( 'link_blank' );
-
 	}
+	edac_update_post_meta( 'link_blank' );
```

Comment:
* Since `edac_update_post_meta` runs regardless of the outcome of the condition, it can be moved outside it
* Replaced `add_option` with `update_option`